src/node_dotenv.h

```diff
@@ -27,7 +31,7 @@ class Dotenv {
   void SetEnvironment(Environment* env);
   v8::Local<v8::Object> ToObject(Environment* env) const;
 
-  static std::vector<std::string> GetPathFromArgs(
+  static std::vector<env_file_data> GetEnvFileDataFromArgs(
```

Comment:
We are inside `Dotenv` class, we don't need EnvFile naming in here:

> Dotenv::GetEnvFileDataFromArgs()
> Dotenv::GetDataFromArgs()

```suggestion
  static std::vector<env_file_data> GetDataFromArgs(
```